data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -951,7 +951,7 @@
 
                 [message]
                     speaker=Elenia
-                    message= _ "Yes, I would... like to join... you Lord Tallin."
+                    message= _ "Yes, I would... like to join... you... Lord Tallin."
```

Comment:
Only spotted after it merged that there are two versions of this string. Are they intentionally different?

If they are, it should probably have a `po` comment.